src/resultsdisassemblypage.ui

```diff
@@ -24,7 +24,14 @@
     <number>0</number>
    </property>
    <item>
-    <widget class="KMessageWidget" name="errorMessage"/>
+    <widget class="KMessageWidget" name="errorMessage">
+     <property name="wordWrap">
+      <bool>false</bool>
```

Comment:
I would think disabling wordwrap could lead to odd sized windows for long error message. can you please enable it?

Reply:
So should I just drop the wordwrap here or explicit enable it or explicit enable it in both (it is explicit disabled in the other place)?

Reply:
in a second patch, explicitly enable it in both

Reply:
If you want that in a separate patch instead of a separate commit: sure, will do that (also would do the second commit here)

Reply:
> furthermore, enabling animations would be easy - that should be just another properly flag, no?

The other parts do it via emitting an event that does this via code https://github.com/KDAB/hotspot/commit/002d0e1b325b738dfbe48c65c9f97f0106ee8b81#diff-bb8df26e05a17485c12aade282ff1bcfe991825bf350fbf75697310a6cc35a25R265-R270 - something I don't feel comfortable to do or adjust currently

Reply:
a patch and a commit is the same, I don't understand how they could be separate? I did not mean a separate merge request like you did not :)

regarding the animation: ok, ignore that for now then.